documentation/src/main/asciidoc/topics/proc_configuring_sql_stores_table.adoc

```diff
@@ -1,16 +1,26 @@
 [id='configuring-sql-cache-stores-table_{context}']
-= Configuring SQL table stores
+= Loading {brandname} caches from database tables
 
-Add SQL table cache stores so {brandname} can preload your caches from database tables.
+Add a SQL table cache store to your configuration if you want {brandname} to load data from a database table.
```

Comment:
We probably want to call out a specific error that Katia has had a lot of fun with.

`ISPN008064: No primary keys found for table AUTHORS, check case sensitivity`

This error currently can mean a few things.

1. The table doesn't actually exist
2. The table name needs to  be made all lower or all upper case (depends upon the DB provider)
3. The table may not have any primary keys defined

Reply:
I'll add this into a SQL store troubleshooting topic but ultimately it'd be good to have these details in a description annotation that we generate as part of an HTML log report. FYI -> https://issues.redhat.com/browse/ISPN-13580